test/garbage_collection_test.js

```diff
@@ -51,7 +47,7 @@ suite('garbage collection tests', function () {
 
     var gc = new GarbageCollector({
       capacity: 1,
-      log: log,
+      log: debug,
```

Comment:
you did `var log = debug` above...  but this is fine too... Basically we'll get the logs if we set `DEBUG` correctly right?